home/components/ProjectTools.tsx

```diff
@@ -51,7 +51,7 @@ function EnabledProjectTools({ pollForUpdates }: Props) {
     (props: State, state: Partial<State>): State => ({ ...props, ...state }),
     initialState
   );
-  const clipboardUpdateInterval = React.useRef<null | number>(null);
+  const clipboardUpdateInterval = React.useRef<null | any>(null);
```

Comment:
Instead of any there should be a specific type for setInterval handles. If you click through to the typedef of setInterval, vs code should show it.

Reply:
The problem is that we have conflicting global ts definitions for the function, one coming from `@types/node` and the other from `@types/react-native`. I'll address in a follow-up.